data/mrqa_urls.txt

```diff
@@ -0,0 +1,19 @@
+# You can use comment to ignore some data sources
+train https://s3.us-east-2.amazonaws.com/mrqa/release/train/SQuAD.jsonl.gz 67afd110c0ad9860c4e88f16a44cd44c
+# train https://s3.us-east-2.amazonaws.com/mrqa/release/train/NewsQA.jsonl.gz d8288b5de5bd10fb42ce5291ef0f7fbe
+# train https://s3.us-east-2.amazonaws.com/mrqa/release/train/TriviaQA-web.jsonl.gz 1d198c0cd60e4d91130e2a2545eb9122
+# train https://s3.us-east-2.amazonaws.com/mrqa/release/train/SearchQA.jsonl.gz fa9c8c6b2f24e4f410cba81ef63ea284
+# train https://s3.us-east-2.amazonaws.com/mrqa/release/train/HotpotQA.jsonl.gz 53e65212b46c74a6ee95e83817443db1
+# train https://s3.us-east-2.amazonaws.com/mrqa/release/train/NaturalQuestionsShort.jsonl.gz f12d2ce98ba0065a79226b9fa22d936a
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/SQuAD.jsonl.gz be0d95e28b470254b3574aeada84a79d
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/NewsQA.jsonl.gz aa9878b7469ad5b5c0f0738636cdb5bd
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/TriviaQA-web.jsonl.gz fdfac306651dd74372f0edcff357ec80
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/SearchQA.jsonl.gz fa087f2cc134f9c316f1d93c40827615
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/HotpotQA.jsonl.gz d0adef52100cbbf93090ba6c06b83b2b
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/NaturalQuestionsShort.jsonl.gz a017834fddfe9df888b7f6cd5bbfba2e
+# dev BioASQ.jsonl.gz http://participants-area.bioasq.org/MRQA2019/ 00ac07599c80c1ac36256f0a2f4bda48
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/DROP.jsonl.gz 0b509e2bebdba16d81f1d3c4f9bedf7c
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/DuoRC.ParaphraseRC.jsonl.gz dcb351275bb7695a54270ca4b59b85c8
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/RACE.jsonl.gz 6cf376f0ab4217e147a2ff4603972e37
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/RelationExtraction.jsonl.gz 20e0fc79f26d13eaac68e62c77980223
+# dev https://s3.us-east-2.amazonaws.com/mrqa/release/dev/TextbookQA.jsonl.gz 3319b8aab7aea550e2392f9b95bbf415
```

Comment:
Do we want to leave all these lines commented? Maybe we can have a separate `mrqa_urls_sample.txt` if it is just for testing purposes or dev purposes?